default/generated/cloud-readiness/08-localhost.windup.yaml

```diff
@@ -64,4 +45,4 @@
   when:
     builtin.filecontent:
       filePattern: .*\.(java|properties|jsp|jspf|tag|xml|txt|yaml)
-      pattern: ws(s)?://((localhost)|(127\.0\.0\.1))+(:[0-9]+)?(/.*)?
+      pattern: ws(s)?://((localhost)|(127\.0\.0\.1))+(:[0-9]+)?(/.*)?
```

Comment:
Q: Why does this appear as a change when the patterns seem identical to the previous version?